dev/WindowsAppRuntime_UniversalBGTaskDLL/Task.idl

```diff
@@ -11,6 +11,5 @@ namespace Microsoft.Windows.ApplicationModel.Background.UniversalBGTask
     runtimeclass Task : Windows.ApplicationModel.Background.IBackgroundTask
     {
         Task();
-        void Run(Windows.ApplicationModel.Background.IBackgroundTaskInstance taskInstance);
```

Comment:
Or do we want to nuke the `[default_interface]` above instead?

Reply:
We could nuke it, ITask doesn't really serve any purpose, but this specific line has to go either way.